app/src/androidTest/java/com/example/karam/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,27 @@
+package com.example.karam;
+
+import android.content.Context;
```

Comment:
what is this file testing ?

Reply:
Nothing. It's a dummy file. I haven't added any tests. I will add it when I add code for business logic.